Automattic/jetpack
# Backup plugin: only redirect from plugins page activation

#### Changes proposed in this Pull Request:

* Plugin activation: Only redirect when activating from Plugins page in the browser

#### Other information:


#### Jetpack product discussion


p9dueE-5CL-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Activate from the CLI and make sure there are no errors `wp plugin activate backup`
* Activate it from the Plugins page in the admin and make sure you are redirected to the Backup plugin page after activation

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202813985335447
